### DISCUSSION TITLE
关于获取群成员等级的问题

### DISCUSSION THREAD

**Discussion by weekdragon** [mamoe/mirai#913] [Q&A] [answered]
我正在从1.3.1升级到2.0,发现之前的lowlevel API的 获取群成员等级方法已经没了，请问现在是有新的方式还是暂时不支持主动获取群成员等级呢

**Comment by sandtechnology** [comment]
> IMirai 低级 API 方法名现在不再带有 _lowLevel 前缀, 直接删除前缀即可。

https://github.com/mamoe/mirai/blob/dev/docs/MigrationFrom1x.md

**Comment by Him188** [comment, marked as answer]
Bot 的一些方法移动到了 IMirai 中
